projects/aca-content/src/lib/components/search/search-input/search-input.component.spec.ts

```diff
@@ -59,6 +70,12 @@ describe('SearchInputComponent', () => {
     fixture.destroy();
   });
 
+  it('should collapse the sidenav when the input value is clicked', () => {
+    spyOn(appServiceMock.appNavNarMode$, 'next').and.callThrough();
```

Comment:
I think you should not use callThrough if possible. This is unit test so you want to just check if next function is called. What happens when appNavNarMode$ is emitted should be tested in files where it is handled. Please remove callThrough if possible because in other case there is risk that unit test will be affected by some other files.